web-console/src/components/timed-button/timed-button.tsx

```diff
@@ -42,7 +42,7 @@ export const TimedButton = React.memo(function TimedButton(props: TimedButtonPro
     label,
     delays,
     onRefresh,
-    type,
+    type: _type,
```

Comment:
`type` is unused, and prefixing with `_` allows it to exist

Reply:
wait, why is this needed? should it just be removed?

Reply:
I think it's specifically pulling it out of the `...other` props to avoid passing it to `<Button>`?